performanceplatform/collector/piwik/base.py

```diff
@@ -0,0 +1,29 @@
+from performanceplatform.utils import requests_with_backoff
```

Comment:
gcloud, ga, and pingdom all have a core.py

webtrends has a base.py

given we are in an inconsistent state, i'm not sure how important this is, but if we went with majority rule this should be called core.py